### ISSUE TITLE
activity diagram: else arrow description is wrong

### ISSUE THREAD

**Issue by jomaway** [hikerpig/pintora#278]
Tried this code inside the live editor:
```
activityDiagram
title: test


if (color?) then (red)
    :print red;
else (green)
    :print green;
endif
```

Expected behavior: print green on the else case.
Problem: else case print's no on the error. See image.
![pintora-output-1712317296657](https://github.com/hikerpig/pintora/assets/22931557/d8e56df5-60de-42ed-ba80-05a11c73b9cd)

Also tested this syntax, but this didn't even compile:
```
activityDiagram
title: test


if (color?) is (red) then
    :print red;
else (green)
    :print green;
endif
```

**Comment by hikerpig** [comment]
I see, it's actually a bug, no matter what label you assign , the label of else branch is always is`no`

**Comment by hikerpig** [comment]
Fixed in v0.7.4
